tests/src/test/scala/scalaz/EitherTTest.scala

```diff
@@ -76,6 +76,10 @@ object EitherTTest extends SpecLite {
     a.flatMap(f andThen EitherT.apply) must_=== a.flatMapF(f)
   }
 
+	"mapF consistent with map" ! forAll { (a: EitherTList[Int, Int], f: Int => String) =>
+		a.map(f) must_=== a.mapF(f andThen (s => Applicative[List].point(s)))
+	}
```

Comment:
please use space instead of tab

Reply:
omg @Jacoby6000 wash your mouth with soap!!!

Reply:
Didn't mean to. The tabs were pasted in from the changes on 7.3 😢